docopt.nimble

```diff
@@ -1,10 +1,10 @@
-version = "0.6.8"
+version = "0.7.0"
```

Comment:
I will not accept a version change in a PR

Reply:
Why not? This is a new feature after all.

Reply:
In my projects (and probably many others) a version change is always done in a commit of itself and by the maintainer.
Not to mention the decision what the actual version should be.

Reply:
Ah I see, I'll revert that then